components/Blockquote/src/index.css

```diff
@@ -0,0 +1,36 @@
+.denhaag-blockquote {
+  border-inline-start: var(--denhaag-blockquote-border-width) var(--denhaag-blockquote-border-style)
+    var(--denhaag-blockquote-border-color, #d5e7d4);
+  padding-block-start: var(--denhaag-blockquote-padding-block-start);
+  padding-block-end: var(--denhaag-blockquote-padding-block-end);
+  padding-inline-start: var(--denhaag-blockquote-padding-inline-start);
+  padding-inline-end: var(--denhaag-blockquote-padding-inline-end);
+}
+
+.denhaag-blockquote__content {
+  margin-block-start: var(--denhaag-blockquote-margin-block-start);
+  margin-block-end: var(--denhaag-blockquote-margin-block-end);
+  margin-inline-start: var(--denhaag-blockquote-margin-inline-start);
+  margin-inline-end: var(--denhaag-blockquote-margin-inline-end);
+}
+
+.denhaag-blockquote__content p {
+  color: var(--denhaag-blockquote-paragraph-color, #1d6b34);
+  font-family: var(--denhaag-blockquote-paragraph-font-family, inherit);
+  font-size: var(--denhaag-blockquote-paragraph-font-size, 1.25rem);
+  font-weight: var(--denhaag-blockquote-paragraph-font-weight);
+  line-height: var(--denhaag-blockquote-paragraph-line-height);
+  margin-block-start: var(--denhaag-blockquote-margin-block-start);
+  margin-block-end: var(--denhaag-blockquote-margin-block-end);
+}
+
+.denhaag-blockquote__attribution {
+  color: var(--denhaag-blockquote-attribution-color, #4b4b4b);
+  font-family: var(--denhaag-blockquote-attribution-font-family, inherit);
+  font-size: var(--denhaag-blockquote-attribution-font-size, 0.625rem);
```

Comment:
I've added a comment in figma about this font-size. I think 10px is not user friendly from an accessibility perspective. https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn?node-id=8123:22417#145088346